schemaregistry/rest_service.go

```diff
@@ -172,10 +173,8 @@ func configureTLS(conf *Config, tlsConf *tls.Config) error {
 		if err != nil {
 			return err
 		}
+		tlsConf.RootCAs = x509.NewCertPool()
 		tlsConf.RootCAs.AppendCertsFromPEM(caCert)
```

Comment:
I think at this point we should also check the return value for this and return an error if we were not able to parse the certificate

```
		if !tlsConf.RootCAs.AppendCertsFromPEM(caCert) {
			return fmt.Errorf("could not parse certificate from %s", caFile)
		}
```